.github/workflows/terragrunt-apply-satellite.yml

```diff
@@ -0,0 +1,104 @@
+name: "Terragrunt apply satellite accounts"
+
+on:
+  workflow_dispatch:
+    # branches:
+    #   - main
+    # paths:
+    #   - ".github/workflows/terragrunt-apply-satellite.yml"
+    #   - "terragrunt/aws/**"
+    #   - "terragrunt/env/**"
+
+env:
+  AWS_REGION: ca-central-1
+  CONFTEST_VERSION: 0.27.0
+  TERRAFORM_VERSION: 1.1.4
+  TERRAGRUNT_VERSION: 0.36.0
+
+permissions:
+  id-token: write
+  contents: read
+  pull-requests: write
+  actions: write
+  checks: write
+  statuses: write
+
+jobs:
+  retrieve-accounts:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Read account file
+        id: set-matrix
+        run: |
+          accounts=$(jq -c "{\"include\": . }" satellite_accounts.json)
```

Comment:
Should this be the same `jq` command that's being used in the `terragrunt-plan-satellite.yml` workflow?

Reply:
💯